tests/ffi/memory/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eux
+
+# shellcheck disable=SC1091
+
+. ../memory/prepare.sh
+
+prepare_test
+reload_config
+
+podman run --rm --replace -d --name ffi-asil \
```

Comment:
another good example of check the exit code.

Reply:
Ack